judge-d-server/src/main/docker/entrypoint.sh

```diff
@@ -1,7 +1,18 @@
-#!/bin/sh
-java \
--XX:+UnlockExperimentalVMOptions \
--XX:+UseCGroupMemoryLimitForHeap \
--XX:MaxRAMFraction=2 \
--XshowSettings:vm \
-$* -jar judge-d.jar
+#!/bin/bash
+
+MEMORY_OPTIONS=" -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=2 -XshowSettings:vm "
+JAVA_OPTIONS=$MEMORY_OPTIONS
+
+if [[ -v VALIDATION_OPTIONS ]]; then
```

Comment:
do we plan to validate VALIDATION_OPTIONS?

Reply:
We can add some validation for VALIDATION_OPTIONS to match a pattern or create a dictionary of acceptable options.

Reply:
I would verify given data if I were you

Reply:
If it would be a big cost (dozens of possible parameters) i think it is not worth considering it.

Reply:
As discussed - we won't do any detailed validation of options as there is a large number of them